test/utils/test_utils.ts

```
@@ -1,7 +1,8 @@
 import test from "node:test";
 import assert from "node:assert";
 
-import { getExtention, settings2GraphAIConfig, deepClean } from "../../src/utils/utils.js";
+import { getExtention } from "../../src/methods/mulmo_media_source.js"
```

ESLint

[error] Insert `;`

(prettier/prettier)